Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp

```diff
@@ -552,52 +552,54 @@ namespace AZ
 
     void ComponentApplication::InitializeSettingsRegistry()
     {
-        SettingsRegistryMergeUtils::ParseCommandLine(m_commandLine);
+        if (m_startupParameters.m_loadSettingsRegistry)
```

Comment:
Uh... InitializeSettingsRegistry is called in the `constructor` and user provided StartupParameters aren't supplied until `ComponentApplication::Create` is called.

Reply:
Changing to your constructor overload suggestion above.

Reply:
Constructor overloading is proving fairly messy due to it also prompting adding constructors across the broader class hierarchy of Applications, such as EditorToolsApplication and EditorQtApplication. Debating cutting out this attempt to include initialization.